src/Files.App/Helpers/ShellContextMenuHelper.cs

```diff
@@ -255,7 +255,15 @@ public static async Task LoadShellMenuItems(string path, CommandBarFlyout itemCo
 					var openWithItem = shellMenuItems.Where(x => (x.Tag as Win32ContextMenuItem)?.CommandString == "openas").ToList().FirstOrDefault();
 					if (openWithItem is not null)
 					{
+						openWithItem.ColoredIcon = new ColoredIconModel()
+						{
+							BaseLayerGlyph = "\uF049",
+							OverlayLayerGlyph = "\uF04A",
+						};
```

Comment:
Oh, color icon is working here?

Reply:
Found out about this and decided to try it

Reply:
It's added in the MenuFlyoutItem before it's converted into an AppBarButton

Reply:
Can we do the same for the "new" flyout? We have a colored icon for the shortcut option but it wasn't working to use the icon there.